python/python/glide/async_commands/redis_modules/search/index.py

```diff
@@ -0,0 +1,80 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+from enum import Enum
+from typing import List, Optional
+
+
+class IndexType(Enum):
+    HASH = "HASH"
+    JSON = "JSON"
+
+
+class Index:
+    """
+    Class for defining an index in a schema.
+
+    Args:
+        index (str): The name of the index.
+        index_type (IndexType): The type of the index, supports HASH (default) and JSON. To index JSON,
+            ensure the RedisJSON module is installed.
+        prefix (Optional[List[str]]): The prefix for the index, tells the index which keys it should index. Multiple prefixes can be added.
+            If not set, the default is '*' (all keys).
+        filter (Optional[str]): A filter expression. It is possible to use "@__key" to access the key that was just added/changed.
+        language (Optional[str]): The default language for documents in the index. Default is English.
```

Comment:
remove Default is English. -> this is a server configuration that might be changed by redis